src/gui/qgsstyleitemslistwidget.cpp

```diff
@@ -541,7 +541,7 @@ void QgsStyleItemsListWidget::openStyleManager()
        || !QgsGui::windowManager()->openStandardDialog( QgsWindowManagerInterface::DialogStyleManager ) )
   {
     // fallback to modal dialog
-    QgsStyleManagerDialog dlg( mStyle, this );
+    QgsStyleManagerDialog dlg( this );
```

Comment:
I think this should be:

    std::unique_ptr<  QgsStyleManagerDialog > dlg;
    if ( mStyle && mStyle != QgsStyle::defaultStyle() )
       dlg = std::make_unique< QgsStyleManagerDialog >( mStyle, this );
    else
       dlg = std::make_unique< QgsStyleManagerDialog >( this );

Otherwise if the `setStyle` method has been called on the widget then the style manager opened won't be for that specified style.